lms/djangoapps/lti_provider/views.py

```diff
@@ -181,3 +185,11 @@ def parse_course_and_usage_keys(course_id, usage_id):
     usage_id = unquote_slashes(usage_id)
     usage_key = UsageKey.from_string(usage_id).map_into_course(course_key)
     return course_key, usage_key
+
+
+def set_user_roles(edx_user, course_key, roles):
+    global_admin = AdminFactory()
+    if 'Staff' in roles or 'Administrator' in roles:
+        auth.add_users(global_admin, CourseStaffRole(course_key), edx_user)
+    if 'Instructor' in roles or 'Administrator' in roles:
```

Comment:
'Instructor' is the only one required for the certification, so I just left 'staff' and 'instructor' for now.